changelog/1.19.0.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "1.19.0"
+description: "Released on 05/19/2021"
+---
+
+### Breaking changes ❗
+
+- infra: Workspace assets created by Coder that were previously located at
+  `/opt/coder`, such as code-server and coder-cli, have been moved to
+  `/var/tmp/coder`
+- infra: Workspaces as code templates have been updated from version _0.1_ to
+  _0.2_. Version _0.1_ templates will no longer work.
```

Comment:
Version 0.1 templates will work though... right?

They just can't be updated before they're migrated I thought

Reply:
I think they *don't* -- @Emyrk can you please confirm?

Reply:
Confirmed that 0.1 templates don't work beginning w/ v1.19

Reply:
@kylecarbs All 0.1 templates currently in use will continue to work (like you can rebuild your workspace). You cannot upload a new 0.1 template.